.github/workflows/php.yml

```diff
@@ -0,0 +1,28 @@
+name: PHP Mexico Bot
+
+on:
+  push:
+    branches: [ master]
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
What about using `php:7.4-cli-alpine` instead of a full Ubuntu? 

All the bot needs is SQLite3 which comes with the image I am using.

Reply:
I think it a great idea, but i do know how to use that image in github actions, may be someone in the comunity can help me with this

Reply:
[A conversation in Slack](https://phpmx.slack.com/archives/C0137RBFWEQ/p1605316332143100) pointed out it is not possible to use other regular Docker images, so this one should be good enough.